src/phantom.js

```diff
@@ -69,41 +70,49 @@ export default class Phantom {
                 this.commands.delete(command.id);
             } else if (message.indexOf('<event>') === 0) {
                 let json = message.substr(7);
-                logger.debug('Parsing: %s', json);
+                this.logger.debug('Parsing: %s', json);
                 const event = JSON.parse(json);
 
                 var emitter = this.events[event.target];
                 if (emitter) {
                     emitter.emit.apply(emitter, [event.type].concat(event.args));
                 }
             } else {
-                logger.info(message);
+                this.logger.info(message);
             }
         });
 
-        this.process.stderr.on('data', data => logger.error(data.toString('utf8')));
-        this.process.on('exit', code => logger.debug(`Child exited with code {${code}}`));
+        this.process.stderr.on('data', data => this.logger.error(data.toString('utf8')));
+        this.process.on('exit', code => this.logger.debug(`Child exited with code {${code}}`));
         this.process.on('error', error => {
-            logger.error(`Could not spawn [${phantomPath}] executable. Please make sure phantomjs is installed correctly.`);
-            logger.error(error);
+            this.logger.error(`Could not spawn [${phantomPath}] executable. Please make sure phantomjs is installed correctly.`);
+            this.logger.error(error);
             this.kill(`Process got an error: ${error}`);
             process.exit(1);
         });
 
         this.process.stdin.on('error', (e) => {
-            logger.debug(`Child process received error ${e}, sending kill signal`);
+            this.logger.debug(`Child process received error ${e}, sending kill signal`);
             this.kill(`Error reading from stdin: ${e}`);
         });
 
         this.process.stdout.on('error', (e) => {
-            logger.debug(`Child process received error ${e}, sending kill signal`);
+            this.logger.debug(`Child process received error ${e}, sending kill signal`);
             this.kill(`Error reading from stdout: ${e}`);
         });
 
         this.heartBeatId = setInterval(this._heartBeat.bind(this), 100);
     }
 
     /**
+     * Returns the winston logger instance used to log messages
+     * @returns {winston}
+     */
+    getLogger() {
```

Comment:
You don't need this anymore. Since `this.logger` already exposes the logger as public